tests/Feature/BerandaPublik/TodayCardLecturingScheduleTest.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Tests\Feature\LecturingSchedule;
+
+use App\Http\Livewire\TodayLecturerCard;
+use App\Models\LecturingSchedule;
+use App\User;
+use Carbon\Carbon;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Livewire\Livewire;
+use Tests\TestCase;
+
+class TodayCardLecturingScheduleTest extends TestCase
```

Comment:
nama classnya kita ganti gini:
```suggestion
class TodayLecturingSchedulesTest extends TestCase
```